com.unity.netcode.gameobjects/Tests/Runtime/PlayerObjectTests.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using Unity.Netcode.TestHelpers.Runtime;
+
+namespace Unity.Netcode.RuntimeTests
+{
+    [TestFixture(HostOrServer.Host)]
+    [TestFixture(HostOrServer.Server)]
+    public class PlayerObjectTests : NetcodeIntegrationTest
+    {
+        protected override int NumberOfClients => 1;
+
+        protected GameObject m_NewPlayerToSpawn;
+
+        public PlayerObjectTests(HostOrServer hostOrServer) : base(hostOrServer) { }
+
+        protected override void OnServerAndClientsCreated()
+        {
+            m_NewPlayerToSpawn = CreateNetworkObjectPrefab("NewPlayerInstance");
+            base.OnServerAndClientsCreated();
+        }
+
+        [UnityTest]
+        public IEnumerator SpawnAnotherPlayerObject()
+        {
+            var originalPlayer = m_PlayerNetworkObjects[1][1];
```

Comment:
I'm trying to rationalize to myself that I should know why I am looking this up by [1][1].  Worth a comment to explain.